src/main/java/chylex/bettersprinting/client/player/impl/LivingUpdate.java

```diff
@@ -64,8 +64,8 @@ else if (player.isPotionActive(MobEffects.NAUSEA) && player.getActivePotionEffec
 		
 		boolean hasAutoJumped = false;
 		
-		if (player.field_189812_cs > 0){
-			--player.field_189812_cs;
+		if (player.autoJumpTime > 0){
+			--player.autoJumpTime;
```

Comment:
I made an assumption here since I was in a hurry - everything tested fine so I assumed it was right but you're the original author and probably know offhand what the original field was.